Keas.Mvc/Services/IdentityService.cs

```diff
@@ -12,6 +12,7 @@
 using Microsoft.EntityFrameworkCore;
 using System.Text;
 using ietws.PPSDepartment;
+using Microsoft.EntityFrameworkCore.Design;
```

Comment:
@jSylvestre what is this doing?

Reply:
@srkirkland Don't know. It isn't used so I created another PR to clean that and a couple other things up pr https://github.com/ucdavis/Keas/pull/531